mne_bids/mne_bids.py

```diff
@@ -386,15 +421,22 @@ def _sidecar_json(raw, task, manufacturer, fname, kind,
     ch_info_json_common = [
         ('TaskName', task),
         ('Manufacturer', manufacturer),
-        ('PowerLineFrequency', powerlinefrequency)]
-    ch_info_json_meg = [
+        ('PowerLineFrequency', powerlinefrequency),
         ('SamplingFrequency', sfreq),
-        ("DewarPosition", "XXX"),
-        ("DigitizedLandmarks", False),
-        ("DigitizedHeadPoints", False),
-        ("SoftwareFilters", "n/a"),
+        ('SoftwareFilters', 'n/a'),
+        ('RecordingDuration', raw.times[-1]),
+        ('RecordingType', rec_type)]
+    ch_info_json_meg = [
+        ('DewarPosition', 'XXX'),
```

Comment:
would you mind changing this to 'n/a'? I know it's not related to your PR, but I just noticed it and it's easy enough for you :)